profiles/arch/sparc/package.use.mask

```diff
@@ -1,6 +1,13 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Sam James <sam@gentoo.org> (2021-03-19)
+# We can't have USE=static on sparc for now because
+# dev-libs/libsigsegv lacks USE=static-libs, and there's
```

Comment:
This explanation is not correct...
It should be something like:
```
# USE=static currently requires USE=-libsigsegv (due to dev-libs/libsigsegv
# not supporting USE=static-libs), but USE=libsigsegv is forced on sparc.
```

Reply:
Again, this isn't intended to be ready for review yet. This is a rough draft. But the explanation isn't wrong - we could however add more information.